integration_tests/pages/refer/reason.ts

```diff
@@ -0,0 +1,43 @@
+import Page from '../page'
+import type { Referral } from '@accredited-programmes/models'
+
+export default class ReasonPage extends Page {
+  referral: Referral
+
+  constructor(args: { referral: Referral }) {
+    super('Add reason for referral and supporting information')
+
+    const { referral } = args
+
+    this.referral = referral
+  }
+
+  shouldContainInformationTypeDetails() {
+    cy.get('[data-testid="information-type-details"]').then(informationTypeDetailsElement => {
+      this.shouldContainDetails(
+        'What type of information can I add?',
+        "you can provide information about why you're referring this person to this programme" +
+          'you can add supporting information that would be useful for the programme team to know' +
+          'you can add any other information that would be useful for the programme team to know',
+        informationTypeDetailsElement,
+      )
+    })
+  }
+
+  shouldContainReasonTextArea() {
+    this.shouldContainTextArea('reason', 'Referral reason and supporting information')
+  }
+
+  shouldContainSaveAndContinueButton() {
+    this.shouldContainButton('Save and continue')
+  }
+
+  submitReason() {
+    const reason = 'This is the reason for the referral'
+    cy.get('[name="reason"]').type(reason)
```

Comment:
Is it worth it/possible to add use a `data-testid` for this? I know we don't on a similar method elsewhere, but it might be good to phase them in with new PRs